README.md

```diff
@@ -19,12 +19,21 @@ Also see [known issues](KNOWN_ISSUES.md) and [CHANGELOG](CHANGELOG.md).
 This plugin is compatible with CSI version 1.3.0.
 
 ### Kubernetes Compatibility
+The following table captures the compatibility matrix of the core filestore driver binary
+`k8s.gcr.io/cloud-provider-gcp/gcp-filestore-csi-driver`
 
 | Filestore CSI Driver\Kubernetes Version | 1.14 | 1.15 | 1.16 | 1.17+ |
 | --------------------------------------- | ---- | ---- | ---- | ----- |
 | v0.2.0 (alpha)                          | yes  |  no  |  no  |  no   |
-| v0.3.1 (beta)                           | no   |  no  |  no  |  yes  |
-| master                                  | no   |  no  |  no  |  yes  |
+| v0.3.1 (beta)                           | no   |  yes |  yes |  yes  |
+| master                                  | no   |  yes |  yes |  yes  |
+
+The manifest bundle which captures all the driver components (driver pod which includes the containers csi-external-provisioner, csi-external-resizer, csi-external-snapshotter, gcp-filestore-driver, csi-driver-registrar;
```

Comment:
maybe give a bit more detail about what's going on, eg saying that not all driver components can be used with all kubernetes versions (eg resizer, snapshotter), and so different overlays must be used to run v0.3.1 on different k8s versions.

Reply:
Done. added an example of snapshotter sidecar.